nservicebus/upgrades/5to6.md

```diff
@@ -234,6 +234,7 @@ The `IMessageMutator` and `IMutateTransportMessages` interfaces are no longer av
 
  * `IncomingContext` and `OutgoingContext` have been replaced with multiple contexts ("stages") per pipeline offering more useful information depending on the requirements. [Pipeline Steps Stages and Connectors](/nservicebus/pipeline/steps-stages-connectors.md) provides more details about the available stages.
  * `Behavior<TContext>` now provides the base class for implementing custom behaviors and replaces `IBehavior<TContext>`.
+ * Behavior instances are created once, when the endpoint starts up, and then cached for the life of the endpoint. This is true even if the behavior type is registered as `InstancePerCall` or `InstancePerUnitOfWork`.
```

Comment:
I strongly believe in removing the APIs now that I see how flawed they are. Let's wait with this PR.